akka-remote/src/main/protobuf/ContainerFormats.proto

```diff
@@ -26,4 +26,23 @@ enum PatternType {
 message Selection {
   required PatternType type = 1;
   optional string matcher = 2;
-}
\ No newline at end of file
+}
+
+message Identify {
+  required Payload messageId = 1;
+}
+
+message ActorIdentity {
+  required Payload correlationId = 1;
+  optional ActorRef ref = 2;
+}
+
+message ActorRef {
```

Comment:
@patriknw - it seems there are a few number of `ActorRef` serialization, i.e. `ActorRefData` in `WireFormats.proto` - do you want me to reuse them, or is it ok to have message distinct to each `.proto` file?